packages/backend/src/models/OrganizationModel.ts

```diff
@@ -75,4 +88,42 @@ export class OrganizationModel {
                 .delete();
         });
     }
+
+    async getAllowedOrgsForDomain(
+        domain: string,
+    ): Promise<UserAllowedOrganization[]> {
+        const rows = await this.database(
+            OrganizationAllowedEmailDomainsTableName,
+        )
+            .whereRaw('? = ANY(email_domains)', domain)
```

Comment:
is this not the same as `whereIn` ?

Reply:
No, whereIn is when the column is a string and you want to check if that value is in an array.
This raw SQL is when the column is a string array and you want to check if your value is in that array.